addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/config/LittleStation.java

```diff
@@ -65,4 +66,13 @@ public String getID() {
     public void setID(String id) {
         this.id = id;
     }
+
+    public boolean isOpen() {
+        return isOpen;
```

Comment:
normally you would name the field open, given the name of the setter and getter.